### PR TITLE
Task/api 428 organization implementation

### PR DESCRIPTION
**Pull request by ilaflamme-va**
This PR is reliant on https://github.freedomstream.io/mulesoft/cdw-schemas/pull/47/files for sentinel tests to pass!

### CODE REVIEW

File: argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+  private void assertSearch(
+      Supplier<Organization.Bundle> invocation, MultiValueMap<String, String> params) {
+    CdwOrganization100Root root = new CdwOrganization100Root();
+    root.setPageNumber(BigInteger.valueOf(1));
+    root.setRecordsPerPage(BigInteger.valueOf(10));
+    root.setRecordCount(BigInteger.valueOf(3));
+    root.setOrganizations(new CdwOrganizations());
+    CdwOrganization xmlPatient1 = new CdwOrganization();
+    CdwOrganization xmlPatient2 = new CdwOrganization();
```
Comment (eclendenning-va) [inline argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java]: not patients

File: argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    CdwOrganization xmlPatient3 = new CdwOrganization();
+    root.getOrganizations()
+        .getOrganization()
+        .addAll(Arrays.asList(xmlPatient1, xmlPatient2, xmlPatient3));
+    Organization patient1 = Organization.builder().build();
+    Organization patient2 = Organization.builder().build();
```
Comment (eclendenning-va) [inline argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java]: also not patients

File: argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationTransformerTest.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    assertThat(tx.addressLine(cdw.address())).isEqualTo(expected.lines());
+  }
+
+  @Test
+  public void telecoms() {
+    assertThat(tx.telecoms(null)).isNull();
+    assertThat(tx.telecoms(new CdwTelecoms())).isNull();
+    assertThat(tx.telecoms(cdw.telecoms())).isEqualTo(expected.telecoms());
+  }
+
```
Comment (eclendenning-va) [inline argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationTransformerTest.java]: could use codings and telecom tests for consistency

File: argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+import org.springframework.util.MultiValueMap;
+
+@SuppressWarnings("WeakerAccess")
+public class OrganizationControllerTest {
+  @Mock MrAndersonClient client;
+
+  @Mock OrganizationController.Transformer tx;
+
+  OrganizationController controller;
+  @Mock HttpServletRequest servletRequest;
```
Comment (bschofield-va) [inline argonaut/src/test/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationControllerTest.java]: Not needed

File: argonaut/src/main/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationController.java [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+public class OrganizationController {
+  private Transformer transformer;
+  private MrAndersonClient mrAndersonClient;
+  private Bundler bundler;
+
+  private Organization.Bundle bundle(
+      MultiValueMap<String, String> parameters,
+      int page,
+      int count,
+      HttpServletRequest servletRequest) {
```
Comment (bschofield-va) [inline argonaut/src/main/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationController.java]: Remove servlet request

File: argonaut/src/main/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationController.java [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+    CdwOrganization100Root root = search(parameters);
+    LinkConfig linkConfig =
+        LinkConfig.builder()
+            .path(servletRequest.getRequestURI())
```
Comment (bschofield-va) [inline argonaut/src/main/java/gov/va/api/health/argonaut/service/controller/organization/OrganizationController.java]: `"Organization"`

**Pull request merged**